etc/packaging/appimages/cartographer-module-aarch64.yml

```diff
@@ -26,58 +26,19 @@ AppDir:
     - arm64
     allow_unauthenticated: true
     sources:
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye main
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian-security bullseye-security main
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye-updates main
-    - sourceline: deb [trusted=yes] https://us-apt.pkg.dev/projects/static-file-server-310021 bullseye main
+    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bookworm main
+    - sourceline: deb [trusted=yes] http://deb.debian.org/debian-security bookworm-security main
+    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bookworm-updates main
+    - sourceline: deb [trusted=yes] https://us-apt.pkg.dev/projects/static-file-server-310021 bookworm main
     include:
-    - libboost-iostreams1.74.0:arm64
-    - libboost-filesystem1.74.0:arm64
-    - libboost-system1.74.0:arm64
-    - libboost-regex1.74.0:arm64
-    - libboost-date-time1.74.0:arm64
-    - libssl1.1:arm64
-    - libgrpc++1:arm64
-    - libgrpc10:arm64
-    - libprotobuf31:arm64
-    - libxcb-render0:arm64
-    - libxcb-shm0:arm64
-    - libabsl20200923:arm64
-    - libc6:arm64
-    - libceres1
-    - libgoogle-glog0v5
-    - libgflags2.2
-    - liblua5.3-0:arm64
-    - libcairo2:arm64
-    - libjpeg62-turbo:arm64
-    - libstdc++6:arm64
-    - libpcl-apps1.11:arm64
-    - libpcl-common1.11:arm64
-    - libpcl-features1.11:arm64
-    - libpcl-filters1.11:arm64
-    - libpcl-io1.11:arm64
-    - libpcl-kdtree1.11:arm64
-    - libpcl-keypoints1.11:arm64
-    - libpcl-ml1.11:arm64
-    - libpcl-octree1.11:arm64
-    - libpcl-outofcore1.11:arm64
-    - libpcl-people1.11:arm64
-    - libpcl-recognition1.11:arm64
-    - libpcl-registration1.11:arm64
-    - libpcl-sample-consensus1.11:arm64
-    - libpcl-search1.11:arm64
-    - libpcl-segmentation1.11:arm64
-    - libpcl-stereo1.11:arm64
-    - libpcl-surface1.11:arm64
-    - libpcl-tracking1.11:arm64
-    - libpcl-visualization1.11:arm64
-    - libx11-6:arm64
-    - libflann1.9:arm64
-    - libxcb1:arm64
-    - libglx0:arm64
-    - libgl1:arm64
-    - libglvnd0:arm64
-    - libnlopt0:arm64
+    - libabsl20220623 
```

Comment:
out of scope for review, but you can probably use a single appimage yml for both platforms

they support environment variable in these, with `${VAR}` syntax I believe, and the diff between these files seems fairly small now

(totally optional, only if it matters to you)

Reply:
We'll handle this in the future, when a larger overhaul of the system occurs